### ISSUE TITLE
einrückungs fehler!

### ISSUE THREAD

**Issue by LFF5644** [mattes2008/ModernCSSDesignPatterns#3]
https://github.com/mattes2008/ModernCSSDesignPatterns/blob/34b76e649a47e4dbf8a07d22b47bc729b5399b8c/preview.html#L9-L13

vor dem `<a>` element kommt TAB

**Comment by L3P3** [comment]
Richtig, denn Einrückung ist keine stilistische Frage (nach Gefühl) sondern eine semantische: Sie zeigt an, was wodrin ist.

**Comment by L3P3** [comment]
@LFF5644 SYNTAX ERROR: Invalid space!

```
einrückungs fehler!
           ^
```

Was ginge:
- Einrückungsfehler
- Einrückungs-Fehler
- Fehler bei der Einrückung

**Comment by LFF5644** [comment]
such dir bessere hobbys @L3P3

**Comment by L3P3** [comment]
Hab ich, ich habe vieles geschafft die letzten Tage!

**Comment by LFF5644** [comment]
DAS MEINE ICH NICHT SONDERN DIE RECHTSCHREIB FEHLER ANDERER ZU BESCHREIBEN!!!

**Comment by LFF5644** [comment]
außerdem ist das richtig es gibt soger google ergebnisse!
![grafik](https://github.com/mattes2008/ModernCSSDesignPatterns/assets/109358910/5d1db3f2-aa66-44f1-8707-32ee4ce68a6d)

**Comment by L3P3** [comment]
https://de.wikipedia.org/wiki/Leerzeichen_in_Komposita

**Comment by L3P3** [comment]
@LFF5644 SYNTAX ERROR: Invalid space!

```
RECHTSCHREIB FEHLER
            ^
```

```
google ergebnisse
      ^
```
See: https://de.wikipedia.org/wiki/Leerzeichen_in_Komposita

-----

ERROR: Unknown word `soger`!
You mean [oger](https://www.google.com/search?q=oger)?

**Comment by LFF5644** [comment]
den artikel hast sicher du erstellt

**Comment by LFF5644** [comment]
ich hoffe du wirst von github gebant für spam issus

**Comment by L3P3** [comment]
reactions.add(xD);

**Comment by L3P3** [comment]
ps ruf an hs hab zeit!
